corehq/motech/forms.py

```diff
@@ -183,7 +183,6 @@ def test_connection_button(self):
                 ),
             ),
             crispy.Div(
```

Comment:
This empty string caused a `KeyError` to be raised when loading the connection settings form. I think the reason was that crispy forms thought this was a field key and so was trying to reference it in the `ConnectionSettings` model. I've tested after removing this empty string and the related CSS classes are still getting added correctly.